R/record-estimation.R

```diff
@@ -224,6 +225,7 @@ estimation_option_names <- list2env(
     "etasample" = "etasamples",
     "etasamples" = "etasamples",
     "etastype" = "etastype",
+    "evalshrink" = "evalshrink",
     "fast" = "fast",
```

Comment:
I know you use NMTRAN to determine which spellings are acceptable, but do you also reference any docs? The new `evalshrink` only has one accepted spelling so just wanted to check that this was the case.

Reply:
As an aside, do you know if there are any plans to update this [repo](https://ghe.metrumrg.com/mrg/nm-help) with NONMEM 7.6 docs?

Reply:
> I know you use NMTRAN to determine which spellings are acceptable, but do you also reference any docs?

I review the docs for things like usage and aliases, but the shortened values aren't documented anywhere that I'm aware of.  I determine those by shortening the option and verifying that the FCON file produced is identical.  (See the script I posted [here](https://github.com/metrumresearchgroup/nmrec/issues/28#issuecomment-1883846886).)

> The new `evalshrink` only has one accepted spelling so just wanted to check that this was the case.

Yes, NM-TRAN doesn't accept any shorter form when I tried.